### PR TITLE
fix: gradient logics

### PR DESCRIPTION
**Pull request by baseplate-admin**

### CODE REVIEW

File: src/lib/components/tippies/profile_dropdown.svelte [hunk 1]
```diff
@@ -50,6 +52,8 @@
             }
         }
     };
+
+    beforeUpdate(() => {});
```
Comment (tokitou-san) [inline src/lib/components/tippies/profile_dropdown.svelte:56]: umm whats with this empty code?

**Review by tokitou-san — APPROVED** [review]

File: src/lib/components/tippies/profile_dropdown.svelte (hunk 1 above)
Comment (baseplate-admin) [reply src/lib/components/tippies/profile_dropdown.svelte:56]: https://github.com/tokitou-san/CoreProject-V3-UI/assets/61817579/2164d1ff-68b2-4db3-bce2-e48382c5c39a

Comment (baseplate-admin) [reply src/lib/components/tippies/profile_dropdown.svelte:56]: I am not merging until work is finished

**Comment by baseplate-admin** [comment]
image

File: src/lib/components/tippies/profile_dropdown.svelte [hunk 2]
```diff
@@ -6,6 +6,8 @@
     import { Avatar } from "@skeletonlabs/skeleton";
     import type { SvelteComponent } from "svelte";
 
+    import { beforeUpdate, onMount } from "svelte";
```
Comment (tokitou-san) [inline src/lib/components/tippies/profile_dropdown.svelte:9]: Where is these being used. I dont see?

Comment (baseplate-admin) [reply src/lib/components/tippies/profile_dropdown.svelte:9]: That's why we add linting to CI

Comment (tokitou-san) [reply src/lib/components/tippies/profile_dropdown.svelte]: Okay? shows error while linting?

**Pull request merged**